content/en/docs/KRE/user/70_features/_index.md

```diff
@@ -27,9 +27,53 @@ func handler(ctx *kre.HandlerContext, data *any.Any) (proto.Message, error) {
  ctx.Logger.Info("[handler invoked]")
 
  req := &Request{}
+ finalRes := &proto.Response{}
+
+ ctx.EarlyReply(finalRes)
+
  res := &EtlOutput{}
 
- ctx.Reply(res)
+ ...
+
+ return res, nil
+
+}
+```
+
+## Early Exit
+
+The early exit feature allows users to end workflow execution when desired.
+
+By calling the function `SetEarlyExit` provided in the `ctx` structure, we are
+instructing the node to reply directly to the entrypoint instead of its following
+workflow node, _only for this execution_.
+
+{{< imgproc early_exit_example Resize "1000x" />}}
+
+Thus, the workflow execution will be stopped without needing to throw exceptions or temper with
+the workflow execution as the following nodes will not be called.
+
+Caution is advised when using this function as users need to remember that the entrypoint can only
+be answered once.
+
+Here is an example in Go:
+
+```go
+func handler(ctx *kre.HandlerContext, data *any.Any) (proto.Message, error) {
+ ctx.Logger.Info("[handler invoked]")
+
+ req := &Request{}
+ res := &EtlOutput{}
+
+ ...
+
+ if conditionExample {
+   ctx.SetEarlyExit()
+   finalRes := &proto.Response{
+    Msg: "Request couldn't be processed"
+   }
+   return finalRes, nil
```

Comment:
The diagrams display the examples as if multiresponse were already deployed, this is a time leap done only in the images because changing the diagrams is harder than the text. However I can agree with having both of them changed to the SendOutput format even though it is not technically correct yet.